eng/common/install-emscripten.cmd

```diff
@@ -0,0 +1,20 @@
+mkdir "%1" 2>nul
```

Comment:
Could you move the files you added in eng/common to eng?

CoreRT repo is not on Arcade yet, but this layout is kind of a prep for that. The runtime repo has explanation: https://github.com/dotnet/runtime/tree/master/eng/common#dont-touch-this-folder